packages/flutter/lib/src/widgets/app.dart

```diff
@@ -1041,12 +1043,53 @@ class _WidgetsAppState extends State<WidgetsApp> with WidgetsBindingObserver {
   }
 
   final Map<LogicalKeySet, Intent> _keyMap = <LogicalKeySet, Intent>{
+    // Next/previous keyboard traversal.
     LogicalKeySet(LogicalKeyboardKey.tab): const Intent(NextFocusAction.key),
     LogicalKeySet(LogicalKeyboardKey.shift, LogicalKeyboardKey.tab): const Intent(PreviousFocusAction.key),
-    LogicalKeySet(LogicalKeyboardKey.arrowLeft): const DirectionalFocusIntent(TraversalDirection.left),
-    LogicalKeySet(LogicalKeyboardKey.arrowRight): const DirectionalFocusIntent(TraversalDirection.right),
-    LogicalKeySet(LogicalKeyboardKey.arrowDown): const DirectionalFocusIntent(TraversalDirection.down),
-    LogicalKeySet(LogicalKeyboardKey.arrowUp): const DirectionalFocusIntent(TraversalDirection.up),
+
+    // Directional keyboard traversal. Not available on web.
+    if (!kIsWeb)
```

Comment:
That each line gets its own `if` reads awkward...

I wonder if this would look nicer with just one if and then the spread operator? Not sure, though.